librz/analysis/rtti_itanium.c

```diff
@@ -762,7 +762,14 @@ static void recovery_apply_vtable(RVTableContext *context, const char *class_nam
 		return;
 	}
 
-	RzAnalysisVTable vtable = { .id = NULL, .offset = 0, .size = 0, .addr = vtable_info->saddr };
+	ut64 startAddress = vtable_info->saddr;
+	RVTableInfo *vtablet = rz_analysis_vtable_parse_at(context, startAddress);
```

Comment:
No need to parse vtables again, you already have vtable info including the size available in the `vtable_info` parameter.